frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/IWithLogger.java

```diff
@@ -0,0 +1,13 @@
+package com.virtuslab.gitmachete.frontend.actions.base;
+
+import com.virtuslab.logger.IEnhancedLambdaLogger;
+
+public interface IWithLogger {
+  IEnhancedLambdaLogger log();
+
+  /**
+   * @return whether the execution on the current thread is in a context where logging should be avoided
+   *         (because e.g. it would lead to a massive spam in the logs)
+   */
+  boolean isLoggingDiscouraged();
```

Comment:
Negative name... didn't have any good idea for a positive (non-negated one)... pls help :pray:

Reply:
`isLoggingPromoted`/`isLoggingEncouraged`?

or (in the reference to https://tools.ietf.org/html/rfc2119 :trollface:)
`shouldLog`